honeybee_vtk/data.py

```diff
@@ -0,0 +1,161 @@
+"""Data json schema and validation."""
+
+import json
+from json.decoder import JSONDecodeError
+from pathlib import Path
+from typing import Dict, List
+from pydantic import BaseModel, validator, Field
+from pydantic.errors import JsonError
+from .model import Model
+from .types import AcceptedValues
+
+
+class Data(BaseModel):
+    """COnfig for data."""
+
+    name: str = Field(
+        description='Name to be give to data. Example, "Daylight-Factor".'
+    )
+
+    object_name: str = Field(
+        description='The name of the model object on which you would like to map this'
+        ' data.'
+    )
+
+    file_paths: List[str] = Field(
+        description='List of paths to the file or files that you are trying to use as'
+        ' data.'
+    )
+
+    @validator('object_name')
+    def validate_object(cls, v: str) -> str:
+        if v in AcceptedValues.names.value:
+            return v
+        else:
+            raise ValueError(
+                f'Object name should be from these {AcceptedValues.names.value}.'
+                f' Instead got {v}.'
+            )
+
+    @validator('file_paths')
+    def validate_paths(cls, v: List[str]) -> List[str]:
+        if all([Path(path).is_file() for path in v]):
+            return v
+        else:
+            raise ValueError(
+                'File paths are not valid.'
```

Comment:
This error message can be much more helpful if you clarify exactly which file(s) is missing.

Reply:
The error message now shows the path that are not valid.